contrib/eclair-cli_autocomplete.sh

```diff
@@ -0,0 +1,62 @@
+# To use this script, source it into your .bashrc
+# Use command `source /path/to/eclair-cli_autocomplete.sh`
```

Comment:
Same as the MempoolCLI PR, please document this file further with links to relevant articles/tutorials, and explain the contents of the scripts.

Reply:
I've tried to keep my comments(explaining code), short in c60a267e3e4d442b29e889e73aa4a25e5e256ed4. If more detailed explanation is needed, please tell I should explain the logic after every 2 line? or after major operations.

Reply:
I've tried to keep my comments(explaining code), short in c60a267e3e4d442b29e889e73aa4a25e5e256ed4. If more detailed explanation is needed, please tell I should explain the logic after every 2 line? or after major operations. 
Also regarding the other commands, shall I wait for the issue to be created(if that's the protocol)  or I can start now?

Reply:
That's enough, no need to add more comments.

> Also regarding the other commands, shall I wait for the issue to be created(if that's the protocol) or I can start now?

You should start now, the issues were only for the beginning to guide you by starting with "simple" commands, now you should implement the remaining ones, don't wait for issues to begin working on this.

Reply:
Also before working on the next PRs, make sure to sync your `master` branch with the repository's `master` branch. You're working based on an obsolete version of `master`, which breaks the github checks (look at the bottom of the page, it still displays the windows build that was removed).

Reply:
See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-command-line

Reply:
And as always, when working on the other commands, don't just try to copy-paste and rush it, make sure you understand what these commands do, this is a good learning exercise for how a lightning node works.

Reply:
> Also before working on the next PRs, make sure to sync your `master` branch with the repository's `master` branch. You're working based on an obsolete version of `master`, which breaks the github checks (look at the bottom of the page, it still displays the windows build that was removed).

I synced the repo's master branch with my fork's master branch.
Then I rebase my branch `auto-comp` (which I am using for auto-completion) on the master branch. 
Still, the windows check is showing.

Reply:
That's strange, that must be a github issue then, feel free to ignore.